src/IRac.cpp

```diff
@@ -3112,6 +3176,13 @@ namespace IRAcUtils {
         return ac.toString();
       }
 #endif  // DECODE_VESTEL_AC
+#if DECODE_VOLTAS
+      case decode_type_t::VOLTAS: {
+        IRVoltas ac(kGpioUnused);
+        ac.setRaw(result->state);
+        return ac.toString();
+      }
+#endif  // DECODE_VOLTAS
```

Comment:
Not related to this issue, but Could this be changed to use `toString()` from `decodeToState()` ?

Reply:
I'm confused by what you mean. Can you please elaborate?

Reply:
Parts of this is duplicated in `decodeToState`  
If ac classes had a common base with `toString` and `toCommon` than these could be reduced to one? and maybe save some binary size as well?

This has nothing to do with this PR, I just happened to see a common pattern that maybe could be reduced?

Reply:
Yes, it's been identified before, and even myself. However, I'm not sure how we can do it easily to reduce code duplication that doesn't potentially increase overall code size. If you've got a theory, please try it out or let me know.